admin/v1/admin.proto

```diff
@@ -607,3 +616,13 @@ message GetWaveFeaturesResponse {
 }
 
 message GetWaveFeaturesRequest {}
+
+// for modifying wave feature default values
+message FeatureSetting {
+  string featurId = 1;
```

Comment:
@capoyon Are you sure you only need the id? Why not include the details so no need to have 2 API calls?